<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DMADialog</class>
 <widget class="QDialog" name="DMADialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>588</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>588</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add / Edit DMA</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>350</y>
     <width>166</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>10</y>
     <width>556</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Description</string>
    </attribute>
    <widget class="QLineEdit" name="dma_number_edit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>196</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>15</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DMA Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>59</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Network</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>14</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Calculated Area [km2]</string>
     </property>
    </widget>
    <widget class="QComboBox" name="network_cbox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>74</y>
       <width>196</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DMA Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dma_name_edit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>75</y>
       <width>196</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="area_edit">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>196</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>dma_number_edit</tabstop>
  <tabstop>dma_name_edit</tabstop>
  <tabstop>area_edit</tabstop>
  <tabstop>network_cbox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DMADialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DMADialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
